cloudflare/cloudflare.py

```
@@ -2366,6 +2485,97 @@ async def scan_url(self, ctx, url: str):
+                        if not scan_data.get("success", False):
+                            continue
+
+                        result = scan_data.get("result", {}).get("scan", {})
+                        verdicts = result.get("verdicts", {})
+                        malicious = verdicts.get("overall", {}).get("malicious", False)
+
+                        if malicious:
+                            await message.delete()
+                            embed = discord.Embed(
+                                title="Cloudflare detected a threat!",
+                                description=f"Cloudflare detected a threat in a message sent in this channel and removed it to safeguard the community.",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)